contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -1093,7 +1093,7 @@ CREATE VIEW sys.sp_databases_view AS
 		sys.babelfish_namespace_ext EXT
 		JOIN sys.babelfish_sysdatabases INT ON EXT.dbid = INT.dbid
 		JOIN pg_catalog.pg_namespace ON pg_catalog.pg_namespace.nspname = EXT.nspname
-		LEFT JOIN pg_catalog.pg_class ON relnamespace = pg_catalog.pg_namespace.oid
+		LEFT JOIN pg_catalog.pg_class ON relnamespace = pg_catalog.pg_namespace.oid where pg_catalog.pg_class.relkind = 'r'
```

Comment:
we should test to cover this change. If there is not test update after this change, it indicates we do not have coverage for this fix

Reply:
Previously, we were accounting virtual relations (views) also for to calculate the database size which is not correct. And anyway, pg_relation_size() will return NULL or zero for such relations. So this changes are made to take only physical table into account to calculate the database size. 

Existing tests for sp_databases_view already covers this case as final database size does not affect with this change.